project/make/test-integration-cli

```diff
@@ -16,7 +16,7 @@ bundle_test_integration_cli() {
 	# even and especially on test failures
 	didFail=
 	if ! {
-		source "$(dirname "$BASH_SOURCE")/.ensure-busybox"
+		source "$(dirname "$BASH_SOURCE")/.ensure-frozen-images"
 		source "$(dirname "$BASH_SOURCE")/.ensure-emptyfs"
```

Comment:
@tianon possibly stupid question, is it deliberate that we put emptyfs after our images?